### DISCUSSION TITLE
502 Bad Gateway nginx/1.15.12 when signing up to Student Developer PAck

### DISCUSSION THREAD

**Discussion by AACypher** [community/community#41095] [GitHub Education]
When signing up the Student Developer Pack, the URL 'https://education.github.com/discount_requests/pack_application' returned a 502 Gateway error. I tried to sign up with the same account on a different device, but it simply returns the same error. Any advice?

**Comment by samus-aran** [comment]
Hi @AACypher 👋 

Sorry you're receiving 502 errors. Are you able to re-visit that page and take any screenshots of when the page fails to load? That would be much appreciated.

**Reply by gonperezramirez** [reply]
![grafik](https://user-images.githubusercontent.com/113845085/208295502-f1a4262b-dfa5-4709-b156-b615e03115b0.png)
Hi there, I'm having the same error. Here is what I see

**Reply by Abhiraj121212** [reply]
The same is happening with me even if I login from a different device, can someone help?

**Reply by Kaspeck** [reply]
It is probably happened after you got in the Student Developer Program. The same thing happened to me, when logged in I am unable to see the benefits page but can still redeem them.

**Reply by thuang45** [reply]
I have the exact same problem!

**Reply by pat-gc** [reply]
me too

**Comment by mohamadreza99** [comment]
I have the same problem. when I am logged out of my github account, this [link](https://education.github.com/pack) opens without any problem. however, when I log in to my account, the 502 Bad Gatewary error appears. it is independent of device or browser.

**Comment by git-ubaid** [comment]
https://education.github.com/ previously the afore mentioned link opened and error occurred when I clicked the "Get Student benefit" button. Now the error occurs on trying to open the link.  Can anybody shed some light.

**Comment by ghost** [comment]
I have the same problem !

**Comment by leonwschr** [comment]
I unfortunately also seem have the same problem.

**Reply by jostin16** [reply]
I had the same problem; I tried different browsers, incognito, and cleaning my cache but those didn't work! I did try to sign in instead of clicking "sign up" [here](https://education.github.com/pack) and it shows that I got the benefits already! I also checked my account subscription on github.com and it shows I have the Pro version!

**Comment by VivekBhand** [comment]
Try clearing cache or open with private/ incognito tabs. It worked for me.

**Reply by leonwschr** [reply]
Doesn’t work for me, not even on a separate device.

**Comment by samus-aran** [comment]
Hi everyone! 👋

Thank you for taking the time to share your experiences. We apologise for the inconveniences you're facing when accessing GitHub Education. The team are aware of the intermittent 502 Bad Gateway errors and are currently working to resolve them. A workaround, that may resolve your issues, is available by visiting https://education.github.com/globalcampus/student. Please let us know if this helps you with navigating GitHub Education platform thereafter.

**Reply by pat-gc** [reply]
it doesnt work either

**Reply by pat-gc** [reply]
oh it does work i had to clear cache

**Reply by gonperezramirez** [reply]
It works, many thanks!
